lib/matplotlib/scale.py

```diff
@@ -448,7 +455,7 @@ class LogitTransform(Transform):
     is_separable = True
     has_inverse = True
 
-    def __init__(self, nonpos):
+    def __init__(self, nonpos='mask'):
```

Comment:
I think this needs changing to 'clip' instead of 'mask'

Reply:
For consistency, I used the same default for LogitTransform (and LogisticTransform) as for LogitScale, which already uses 'mask'.

Reply:
👍